### DISCUSSION TITLE
[Feature] istio skywalking envoyAccessLogService.address only k8s service name?

### DISCUSSION THREAD

**Discussion by ccwxl** [apache/skywalking#9153] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

```
istioctl manifest apply \
  --set profile=demo \
  --set meshConfig.enableEnvoyAccessLogService=true \
  --set meshConfig.defaultConfig.envoyAccessLogService.address=<skywalking-oap.skywalking.svc:11800>
```
- `<skywalking-oap.skywalking.svc:11800>` Can it only be the service name of k8s? Can it be an external address?

### Use case

- istio skywalking integrated.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
From a tech perspective, it could be any accessible IP address. In practice, it depends on what analysis do you try to activate? In some cases, SkyWalking OAP server needs to be inside k8s to access API server to read metadata.

**Reply by ccwxl** [reply]
I try to understand, 
![image](https://user-images.githubusercontent.com/13701232/171336157-2a65de83-5bbb-4e85-abe1-9f3819f92792.png)
1.  For `Observe Service Mesh ` AccessLog data flows is `envoy-->satellite(convert ip to real name,)-->oap collector `,internal address need access k8s?
2. For `Observe Control Plane (Istio)` metrics, access logs, tracing use Telemetry Api  data flows is `envoy-->otel-->oap collector `, if it is `Prometheus` `otel` change to `Prometheus Receiver`,skywalking can be an external address. ?
3. For `Observe Data Plane (Envoy)`   with istio. Set the envoy envoyMetricsService address through istio, skywalking can be an external address. ?

**Reply by wu-sheng** [reply]
Read these two, you may get a full understanding.
1. https://skywalking.apache.org/blog/2020-12-03-obs-service-mesh-with-sw-and-als/
2. https://skywalking.apache.org/blog/obs-service-mesh-vm-with-sw-and-als/
